service/src/vsids.rs

```diff
@@ -46,18 +48,13 @@ use std::sync::{Arc, RwLock};
 // ensuring the newer edges retain more relevance.      //
 //////////////////////////////////////////////////////////
 
-type Edge = (String, String, String);
-type SourceKey = (String, String);
-type Weight = f64;
-pub type NodeId = usize;
-
 #[derive(Debug)]
 pub enum GraphOp {
   SetEdge {
     context: String,
```

Comment:
It is still better to designate `context` as a separate type, such as `ContextName` (set to string). That way, the semantics will bear even more info, and there is even less chance of confusion.